src/miner_util.erl

```diff
@@ -90,7 +94,14 @@ metadata_fun() ->
         case application:get_env(miner, mode, gateway) of
             validator ->
                 Vsn = element(2, hd(release_handler:which_releases(permanent))),
-                Map#{<<"release_version">> => list_to_binary(Vsn)};
+                FWRelease = case filelib:is_regular(?LSB_FILE) of
```

Comment:
this is the code that should replace the gateway section below, this section was fine as-is